internal/mailer/validate.go

```diff
@@ -61,6 +61,14 @@ var invalidHostMap = map[string]bool{
 	"email.com":     true,
 }
 
+var blockedMXRecords = map[string]bool{
```

Comment:
These should live in config like your other pull request. I know others mentioned the list should be small and since it is for now config is okay.

But the blockedMXRecords map should be attached to the EmailValidator struct, when you create the struct in `newEmailValidator` you can build the map above, like we do `serviceHeaders map[string][]string`.